lib/internal/make-api-call.js

```diff
@@ -93,6 +98,7 @@ exports.inject = function(options) {
       until: function(response) {
         return !(
             response == null
+            || response.status === 403  // Geolocation API only.
```

Comment:
Also, I didn't understand this bit. It looks like it happens for all APIs, not just geolocation. And we shouldn't retry for 403 generally.